website/content/en/preview/concepts/nodepools.md

```diff
@@ -71,8 +71,18 @@ spec:
       # The amount of time a Node can live on the cluster before being removed
       # Avoiding long-running Nodes helps to reduce security vulnerabilities as well as to reduce the chance of issues that can plague Nodes with long uptimes such as file fragmentation or memory leaks from system processes
       # You can choose to disable expiration entirely by setting the string value 'Never' here
+
+      # Note, changing this value in the nodepool will drift the nodeclaims. 
       expireAfter: 720h | Never
 
+      # The amount of time that a node can be draining before it's forcibly deleted. A node begins draining when it's deleted. Pods with 
```

Comment:
```suggestion
      # The amount of time that a node can be draining before it's forcibly deleted. A node begins draining when a delete call is made against it, starting its finalization flow. Pods with 
```

This may not be the right wording, but I'd say something slightly different since this statement here feels like it contradicts the statement before it